src/main.cc

```diff
@@ -136,4 +137,9 @@ BENCHMARK_TEMPLATE(BM_FAST_DESERIALIZE_AND_TRAVERSE, cereal_bench)
 BENCHMARK_TEMPLATE(BM_FAST_DESERIALIZE_AND_TRAVERSE, capnp_bench)
     ->Unit(benchmark::kMillisecond);
 
-BENCHMARK_MAIN();
\ No newline at end of file
+BENCHMARK_TEMPLATE(BM_SERIALIZE, zpp_bits_bench)
+    ->Unit(benchmark::kMillisecond);
+BENCHMARK_TEMPLATE(BM_SAFE_DESERIALIZE, zpp_bits_bench)
+    ->Unit(benchmark::kMillisecond);
```

Comment:
where's traverse + deserialize, etc.?

Reply:
Added them in the appropriate places- In total there are 3 - serialize, deserialize, and traverse. Not sure if the deserialize and traverse together is necessary to benchmark in this case.

Reply:
As this is a very common use case (first deserialize, then traverse the data), I think this is a benchmark with high practical impact.

Reply:
Added it, it came out as a sum of deserialize and traverse separately